.mk/docker.mk

```diff
@@ -1,20 +1,17 @@
-DOCKER_PASSWORD ?= $(shell echo "${DOCKER_PASSWORD_ENCODED}" | base64 -d)
-DOCKER_HOSTNAME ?= ghcr.io
-DOCKER_NAMESPACE ?= the-mesh-for-data
+#export DOCKER_USERNAME ?= ""
+#export DOCKER_PASSWORD ?= ""
```

Comment:
```
export DOCKER_USERNAME ?=
export DOCKER_PASSWORD ?=
```

else it will stop supporting private repos (which may be used for tests)

Reply:
will change, thanks. 

Regarding your comment "which may be used for tests", note the issue that this PR tries to resolve. We will only be able to do such tests in master builds, not in PRs.